akka-actor-typed/src/main/scala/akka/actor/typed/delivery/internal/ChunkedMessage.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.delivery.internal
+
+import akka.annotation.InternalApi
+import akka.util.ByteString
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[akka] final case class ChunkedMessage(
+    serialized: ByteString,
+    firstChunk: Boolean,
+    lastChunk: Boolean,
```

Comment:
no chunk Id?

Reply:
since it's based on the sequence numbers from reliable delivery there is no need for more than first and last markers